app/assets/vue/views/Home.vue

```diff
@@ -83,4 +158,9 @@ export default {
 .contact {
   margin-top: 15%;
 }
+
+.user-datas-div {
+  height: 50vh;
+  overflow-y: scroll;
```

Comment:
Ouf.... really not sure about the UX on mobile here.

Reply:
Is the problem the size or the scroll?
The height may indeed be too small on the phone.